web-storybook/.storybook/preview.jsx

```diff
@@ -2,11 +2,14 @@ import React from 'react';
 import { INITIAL_VIEWPORTS } from '@storybook/addon-viewport';
 import { CssBaseline } from '@mui/material';
 import { BrowserRouter as Router } from 'react-router-dom';
+import { Buffer } from 'buffer';
 
 import ThemeProvider from 'web-components/src/theme/RegenThemeProvider';
 import 'web-components/src/theme/fonts.css';
 import '../../tailwind.css';
 
+window.Buffer = Buffer;
```

Comment:
is this fixing the buffer issue?

Reply:
yes. Buffer is a Node global so it's not available by default in a browser environment